### DISCUSSION TITLE
Support more date formats

### DISCUSSION THREAD

**Discussion by DameGAD** [navidrome/navidrome#2493] [Ideas]
Navidrome can read the "YYYY-MM-DD" format, but neither "YYYY.MM.DD" nor "YYYYMMDD" (without any separator) are correctly read. It would be nice if Navidrome could read them too.

[edit] I just came across files tagged with spaces ("YYYY MM DD"), it could be another format to consider.

**Comment by certuna** [comment]
Only YYYY-MM-DD is allowed in the tagging standards, unfortunately. Probably best to fix them before feeding them to Navidrome.
